src/components/login/LoginForm.js

```diff
@@ -29,13 +30,20 @@ const LoginForm = (props) => {
                     value ={password}
                     onChange={passwordChange}
                 />
-                <label className="active" for="password">Password</label>
+                <label>Password</label>
+                {/* <label for="password">Password</label> */}
             </div>
-            <input type="submit" value="Login"
+            <Loader loaded={!isLoading}>
+                <input type="submit" value="Login"
                 className="waves-effect waves-light btn-small login"
                 onClick = {handleSubmit}
                 />
-            <Link to='/password-reset' className="forgot-password">Forgot Password?</Link>
```

Comment:
@akram256 by removing this line you are removing the RESET password functionality, please fix this because it shall cause issues during the demo.